Dockerfile

```
@@ -0,0 +1,26 @@
+FROM python:3.8-slim-buster AS prepare
```

Can we use bullseye already?
```suggestion
FROM python:3.8-slim-bullseye AS prepare
```